### PR TITLE
docs: rename example filename

### PR DESCRIPTION
**Pull request by RomainGoussault**
Signed-off-by: Romain Goussault <romain.goussault@owkin.com>

### CODE REVIEW

File: examples/titanic_example/run_titanic.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 # Import all the dependencies
 # ---------------------------
 
-# sphinx_gallery_thumbnail_path = 'auto_examples/titanic_example/images/thumb/sphx_glr_plot_titanic_thumb.jpg'
+# sphinx_gallery_thumbnail_path = 'auto_examples/titanic_example/images/thumb/sphx_glr_run_titanic_thumb.jpg'
```
Comment (ThibaultFy) [inline examples/titanic_example/run_titanic.py:42]: Can't we completely remove this line ?

**Review by ThibaultFy — COMMENTED** [review]
Thx, and nice catch for changing the pattern :) 
Plz update the changelog and the commit name !

File: examples/titanic_example/run_titanic.py (hunk 1 above)
Comment (RomainGoussault) [reply examples/titanic_example/run_titanic.py]: yes, done

**Review by ThibaultFy — APPROVED** [review]
thx !

**Pull request merged**